examples/Receiver_Response.py

```diff
@@ -0,0 +1,164 @@
+)
+# $\nu_{test}$
+nu_test_kHz = np.array([float(j.split("_")[1]) for j in file2_nodes])
+# {{{ Calculate input power (acquired on Oscilloscope)
+# {{{ Make empty nddata to drop the calculated $V^{2}$ into with corresponding
+#     frequency ($\nu$) output by AFG source, and set the frequencies based on
+#     the node names
+control = psd.ndshape([len(control_frqs_kHz)], ["nu_test"]).alloc()
+control.setaxis("nu_test", control_frqs_kHz).set_units("nu_test", "Hz")
+# }}}
+for j, nodename in enumerate(file1_nodes):
+    rf_frq = control["nu_test"][j]
+    d = psd.find_file(
+        file1,
+        expno=nodename,
+        exp_type=data_dir,
+    )
+    # {{{ fit signal in t domain to complex exponential
+    A, omega, phi, t = symbols("A omega phi t", real=True)
+    f = psd.lmfitdata(d)
+    f.functional_form = A * sp.exp(1j * 2 * np.pi * omega * t + 1j * phi)
+    f.set_guess(
+        A=dict(value=5e-2, min=1e-4, max=1),
+        omega=dict(
+            value=rf_frq,
+            min=rf_frq - 1e4,
+            max=rf_frq + 1e4,
+        ),
+        phi=dict(value=0.75, min=-np.pi, max=np.pi),
+    )
+    f.fit(use_jacobian=False)
+    fit = f.eval()
+    # }}}
+    V_amp = f.output("A")
+    control["nu_test", j] = V_amp**2
+# {{{ make spline for power going into RX box
+control.sort("nu_test")
+Pin_spline = control.spline_lambda()
+# }}}
+# }}}
+# {{{ Calculate $dg^2$
+for j, nodename in enumerate(file2_nodes):
+    d = psd.find_file(
+        file2,
+        exp_type=data_dir,
+        expno=nodename,
+    )
+    carrier = (
+        d.get_prop("acq_params")["carrierFreq_MHz"] * 1e6
+    )  # $\nu_{RX,LO}$
+    if j == 0:
+        # Allocate and array that's shaped like one of the datasets
+        # acquired (with 100 captures and a direct t dimension)
+        # but add axis to store the frequency of the test signal
+        rec_data = (d.shape + ("nu_test", len(nu_test_kHz))).alloc()
+        rec_data.setaxis("t", d.getaxis("t")).set_units("t", "s")
+        rec_data.setaxis("nu_test", nu_test_kHz).set_units("nu_test", "Hz")
+    rec_data["nu_test", j] = d
+    SW = str(d.get_prop("acq_params")["SW_kHz"]) # For labeling final plot
+rec_data.sort("nu_test")
+rec_data.rename("nScans", "capture")  # To be more consistent with the
+#                                        oscilloscope data rename the nScans
+#                                        dimension
+acq_time = np.diff(rec_data.getaxis("t")[np.r_[0, -1]]).item()
+# {{{ Calculate PSD for each frequency (we will calculate power from the A
+#     of the convolved test signal)
+rec_data.ft("t", shift=True)  # $dg\sqrt{s/Hz}$
+rec_data = abs(rec_data) ** 2  # $dg^{2}*s/Hz$
+rec_data.mean("capture")
+rec_data /= acq_time  # $dg^{2}/Hz$
+# Convolve using $\lambda_{G}$ specified above
+rec_data.convolve("t", lambda_G, enforce_causality=False)
+rec_data.ift("t")
+rec_data.run(np.conj)  # Empirically needed to give offset
+#                        that increases with field
+rec_data.ft("t")
```

Comment:
this is super weird -- why would you not just do this before the ft, rather than moving to fourier domain and then back again.